engine-extensions/src/main/java/io/nosqlbench/engine/extensions/computefunctions/ComputeFunctions.java

```diff
@@ -85,10 +90,11 @@ public static double precision(long[] relevant, long[] actual) {
 
     public static double precision(long[] relevant, long[] actual, int k) {
         if (actual.length < k) {
-            throw new RuntimeException("indices fewer than limit, invalid precision computation: index count=" + actual.length + ", limit=" + k);
+            logger.warn("Returned indices fewer than limit in recall calculation: index count=" + actual.length + ", limit=" + k);
```

Comment:
same comment as above for all these logger.warn

Reply:
same